docs/projects/example-projects.team.md

```diff
@@ -0,0 +1,69 @@
+![](/docs/assets/projects-team.png)
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <title>Document</title>
+    <style>
+        .author {
+            font-size: 0.9em;
+        }
+        
+        .title {
+            font-size: 1.1em;
+            font-weight: 500;
+            border-bottom: 1px dotted black;
+        }
+        
+        .sans {
+            font-family: Lato, sans-serif;
+            color: rgba(0, 0, 0, 0.87);
+        }
+        
+        .key {
+            font-weight: 700;
+        }
+        
+        ul {
+            padding: 5px;
+        }
+        
+        li {
+            list-style: none;
+        }
+    </style>
+</head>
+
+<body>
+    <h1 class="title sans">Projects.team example</h1>
+    <ul id="project"></ul>
+    <script>
+        const projectOutput = document.getElementById('project')
+
+        function formatObject(obj) {
```

Comment:
This function is really hard to understand, maybe . Also, what will the output be? Wouldn't it be better to use `forEach` on `members` as we know it will be array of objects and then make Object.keys?

Reply:
I created it by this way because I want to keep this example consistent with other examples in docs.
Futhermore if our endpoint will be changed (e.g. array will be wrapped into object with some extra data) the example will be still valid.

Reply:
Okey then :+1: